starport/services/chain/serve.go

```diff
@@ -459,14 +459,19 @@ func (c *Chain) start(ctx context.Context, conf conf.Config) error {
 	c.served = true
 
 	// print the server addresses.
-	fmt.Fprintf(c.stdLog(logStarport).out, "🌍 Running a Cosmos '%[1]v' app with Tendermint at %s.\n", c.app.Name, xurl.HTTP(conf.Servers.RPCAddr))
-	fmt.Fprintf(c.stdLog(logStarport).out, "🌍 Running a server at %s (LCD)\n", xurl.HTTP(conf.Servers.APIAddr))
+	fmt.Fprintf(c.stdLog(logStarport).out, "🌍 Running a Cosmos '%[1]v' app with Tendermint at %s.\n", c.app.Name, xurl.HTTP(conf.Host.RPC))
+	fmt.Fprintf(c.stdLog(logStarport).out, "🌍 Running a server at %s (LCD)\n", xurl.HTTP(conf.Host.API))
+
+	host := conf.Faucet.Host
+	if conf.Faucet.Port != 0 {
+		host = fmt.Sprintf("0.0.0.0:%d", conf.Faucet.Port)
```

Comment:
```suggestion
		host = fmt.Sprintf(":%d", conf.Faucet.Port)
```

Let's follow the same pattern here:
https://github.com/tendermint/starport/blob/e94b428a1c4856cf101463d9afc8d8d1cec632a8/starport/chainconf/config.go#L24-L30